docs/howto/custom-operator.rst

```diff
@@ -60,15 +60,24 @@ You can now use the derived custom operator as follows:
 
 .. code:: python
 
-    hello_task = HelloOperator(task_id='sample-task', dag=dag, name='foo_bar')
+    from custom_operator.hello_operator import HelloOperator
+
+    with dag:
+        hello_task = HelloOperator(task_id='sample-task', name='foo_bar')
+
+**Note**: For imports to work, you should place the file in a directory that
```

Comment:
This note should be against the previous snippet, not the dag one :)

Oh and also this should be `.. note::` https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#colored-boxes-note-seealso-todo-and-warnings